packages/react-fluentui-examples/List/list-active-element/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "react-fluentui-examples-list-active-element",
```

Comment:
There's no build infra in the contrib repro to handle application type projects, as all the infra is built around publishable packages. The task runner (NX) isn't even integrated to this package and it maintains its own dependencies.

I'd suggest to move it out to another `/apps` folder on the root to keep the separation from the publishable packages